SpatialGDK/Build/Scripts/FindMSBuild.bat

```diff
@@ -0,0 +1,25 @@
+rem Convenient code to find MSBuild.exe from https://github.com/microsoft/vswhere 
+rem We only support VS2017 so only look for MS_Build 15.0 

```

Comment:
We support 2015 :(
https://docs.improbable.io/unreal/alpha/content/get-started/dependencies

Reply:
2015 has been removed from the docs :D

Reply:
(For the next release)